ql/src/java/org/apache/hadoop/hive/ql/exec/tez/monitoring/TezJobMonitor.java

```diff
@@ -513,4 +527,12 @@ ProgressMonitor progressMonitor(DAGStatus status, Map<String, Progress> progress
     }
     return TezProgressMonitor.NULL;
   }
+
+  public String getSummary() {
```

Comment:
do we need all these console wrapper methods, why not expose the Console object and operate on it?

Reply:
hm, let me check, makes sense